src/components/NavBarDrawer.css

```diff
@@ -1,3 +1,9 @@
 img {
   height: 1.2em;
 }
+button {
+  align-items: center;
+}
+.MuiListItemIcon-alignItemsFlexStart {
```

Comment:
Is this class being used anywhere? I couldn't see an effect after removing this.